docs/static/staticwebapp.config.json

```diff
@@ -4,11 +4,6 @@
 			"route": "/docs/fcdocstest",
 			"redirect": "/docs/apis",
 			"statusCode": 301
-		},
-		{
-			"route": "/docs/apis/*",
-			"redirect": "/docs/apis",
```

Comment:
I thought what I had seen here before was redirecting from `/docs/apis/*` to `/docs/api/v1` or something like that? Was that not the intention with the redirects? I guess since we don't expect to add any more routes under `apis/<something>`, redirecting all those to their corresponding `api/v1/<something>` would be ok?

Reply:
This change was to test a couple of things; we won't push out the `v1` paths until we have the changes to structure the documentation according to that also (that will be part of a separate PR with the routes updated)